tests/helpers.py

```diff
@@ -1572,3 +1572,17 @@ def wait_for_ct_pod_recovery():
         else:
             return False
     return True
+
+
+def label_worker_node(node_list, label):
+    """
+    Function to label worker node for running app pods on specific worker nodes.
+
+    Args:
+        node_list (list): List of node name
+        label (str): Label to be added in worker
+    """
+    ocp_obj = OCP()
+    for node in node_list:
+        out = ocp_obj.exec_oc_cmd(command=f"label node {node} nodetype={label}", out_yaml_format=False)
```

Comment:
I feel this function's second arguments should be generic (k,v), because here you are specifically passing the value for 'nodetype' attribute and calling this function as label_worker_node, what if I want to set a different label i.e. (k,v) on this node ? 

Correct me if that is not the intended use of this function.

Reply:
+1 label should have 'key' and 'value' to be generic.

Reply:
ack

Reply:
Updated function to get label_key and label_value from user, resolving this comment.